module/journal.js

```diff
@@ -32,6 +32,13 @@ export default class WorldAnvilBrowser extends Application {
    */
   _displayWIP = true;
 
+  /**
+   * Storing which categories were shrinked
+   * @type {string[]}
+   * @private
+   */
+  _shrinkedCategories = [];
```

Comment:
We typically use "expanded" vs "collapsed" for this concept rather than "shrinked", it would be good to standardize naming in that way.